homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -226,14 +226,47 @@ def get(self, request, username):
             "config": {
                 "mac": "00:00:00:00:00:00",
                 "swversion": "01003542",
+                "apiversion": "1.17.0",
                 "whitelist": {HUE_API_USERNAME: {"name": "HASS BRIDGE"}},
                 "ipaddress": f"{self.config.advertise_ip}:{self.config.advertise_port}",
+                "linkbutton": True,
             },
         }
 
         return self.json(json_response)
 
 
+class HueConfigView(HomeAssistantView):
+    """Return config view of emulated hue."""
+
+    url = "/api/{username}/config"
+    name = "emulated_hue:username:config"
+    requires_auth = False
+
+    def __init__(self, config):
+        """Initialize the instance of the view."""
+        self.config = config
+
+    @core.callback
+    def get(self, request, username):
+        """Process a request to get the configuration."""
+        if not is_local(request[KEY_REAL_IP]):
+            return self.json_message("only local IPs allowed", HTTP_UNAUTHORIZED)
+        if username != HUE_API_USERNAME:
+            return self.json(UNAUTHORIZED_USER)
+
+        json_response = {
+            "mac": "00:00:00:00:00:00",
+            "swversion": "01003542",
+            "apiversion": "1.17.0",
+            "whitelist": {HUE_API_USERNAME: {"name": "HASS BRIDGE"}},
+            "ipaddress": f"{self.config.advertise_ip}:{self.config.advertise_port}",
```

Comment:
I'm not sure this will work since `CONF_ADVERTISE_IP ` and `CONF_ADVERTISE_PORT` are optional.

Reply:
```
        # Calculated effective advertised IP and port for network isolation
        self.advertise_ip = conf.get(CONF_ADVERTISE_IP) or self.host_ip_addr

        self.advertise_port = conf.get(CONF_ADVERTISE_PORT) or self.listen_port


```

Looks like we always set it so should be fine